src/platform-includes/sourcemaps/overview/javascript.mdx

```diff
@@ -2,6 +2,14 @@
 
 We've compiled a list of guides on how to upload source maps to Sentry for the most popular JavaScript build tools:
```

Comment:
```suggestion
```

We should either move this sentence or just get rid of it.

Reply:
I'll remove it, good catch 👍